src/mixins/amalgamation-mixin.ts

```diff
@@ -192,21 +201,22 @@ export default class AmalgamationMixin extends Vue {
   }
 
   /**
-   * Get the business information, mailing address, email, and first filing if in LEAR.
-   * Otherwise, return error.
+   * Fetches the business' auth information, business info, addresses, first task, and first filing.
+   * @param identifier The business identifier.
    */
-  async fetchAmalgamatingBusinessInfo (item: any): Promise<any> {
-    // Get the auth info, business info, addresses and filings concurrently.
+  async fetchAmalgamatingBusinessInfo (identifier: string): Promise<any> {
+    // Make all API calls concurrently without rejection.
     // NB - if any call failed, that item will be null.
-    const [ authInfo, businessInfo, addresses, firstFiling ] =
+    const [ authInfo, businessInfo, addresses, firstTask, firstFiling ] =
       await Promise.allSettled([
-        AuthServices.fetchAuthInfo(item.identifier),
-        LegalServices.fetchBusinessInfo(item.identifier),
-        LegalServices.fetchAddresses(item.identifier),
-        LegalServices.fetchFirstOrOnlyFiling(item.identifier)
+        AuthServices.fetchAuthInfo(identifier),
+        LegalServices.fetchBusinessInfo(identifier),
+        LegalServices.fetchAddresses(identifier),
+        LegalServices.fetchFirstTask(identifier),
+        LegalServices.fetchFirstOrOnlyFiling(identifier)
       ]).then(results => results.map((result: any) => result.value || null))
 
-    return { authInfo, businessInfo, addresses, firstFiling }
+    return { authInfo, businessInfo, addresses, firstTask, firstFiling }
   }
```

Comment:
I thought it was better to pass in `identifier` instead of an untyped business object in which we "hope" there is an identifier. Identifier is the only thing we need from the business in this function.